shentu/chain.json

```diff
@@ -36,18 +36,18 @@
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/ShentuChain/mainnet/main/shentu-2.2/genesis.json"
     },
-    "recommended_version": "v2.8.0",
```

Comment:
Please retain v2.8.0 inside the versions array

Reply:
Added all history versions in versions array field